### ISSUE TITLE
"horizontal-stack" has no effect on line break

### ISSUE THREAD

**Issue by eufingdi** [UI-Lovelace-Minimalist/UI#445]
Hello,

I have the problem that in the dashboard layout cards are placed to the right of other elements (like cards or chips) instead of - as expected by me with the command "horizontal-stack" - in a new row below the previous cards. 

Does anyone have an idea what I am doing wrong or even a solution for the problem already?

Many thanks and greetings from Dirk

![Screenshot_20220219-074448_Chrome](https://user-images.githubusercontent.com/70875682/154790022-b3bf1064-77a2-4401-9460-35325949476b.jpg)

![Screenshot_20220219-075047_Chrome](https://user-images.githubusercontent.com/70875682/154790244-12579c95-4894-4cc6-9e15-789e1d421e22.jpg)
![Screenshot_20220219-075247_Chrome](https://user-images.githubusercontent.com/70875682/154790279-929ca0b8-dd6a-434b-82fc-ad986b5ae1d1.jpg)

**Comment by basbruss** [comment]
To put all horizontal-stacks in one column you have to put them all within a vertical-stack :wink:. Otherwise it will auto align in multiple columns on bigger screen sizes.

**Comment by eufingdi** [comment]
Hello Basbruss,
I had already tried that, but I had indented the following entries one level further. That was the error. Now the following "horizontal stack" is on the same level as the "vertical stack" and it works great.

Thank you very much for your tip, which gave me the solution! :-)

![Screenshot 2022-02-19 085126](https://user-images.githubusercontent.com/70875682/154792142-14745632-8770-4a8e-bbbb-72f1360a0cd4.jpg)
